src/wallet/rpcwallet.cpp

```diff
@@ -3932,17 +3932,17 @@ UniValue sethdseed(const JSONRPCRequest& request)
 
     LegacyScriptPubKeyMan& spk_man = EnsureLegacyScriptPubKeyMan(*pwallet);
 
-    if (pwallet->chain().isInitialBlockDownload()) {
```

Comment:
Perhaps make this change in the last commit where the other removals of `isInitialBlockDownload` take place? 

IIUC, the commit is misnamed and could better be called something like "Remove Chain::isInitialBlockDownload`